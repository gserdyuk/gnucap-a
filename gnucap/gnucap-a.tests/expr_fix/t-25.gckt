* 

.param u='defined(1)'
.param v='defined(0)'

.param a='1'
.param w='defined(a)'
.param x='defined(b)'

.param y='defined(a+1)'
.param z='defined(b+1)'
.param zz='defined(a+b+1)'

v1 1 0 dc 'u'
v2 2 0 dc 'v'
v3 3 0 dc 'w'
v4 4 0 dc 'x'
v5 5 0 dc 'y'
v6 6 0 dc 'z'
v7 7 0 dc 'zz'

.print op v(v*)
.op
.end

 