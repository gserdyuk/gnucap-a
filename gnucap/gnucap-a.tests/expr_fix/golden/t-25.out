Gnucap gnucap-arails  2009.12.07 RCS 26.136 globalpar
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
* 
#           v(v1)      v(v2)      v(v3)      v(v4)      v(v5)      v(v6)      v(v7)     
 27.        1.         1.         1.         0.         1.         0.         0.        
